### ISSUE TITLE
cannot import name 'StableVideoDiffusionPipeline' from 'diffusers'

### ISSUE THREAD

**Issue by hzxie** [VITA-Group/4DGen#7]
```
Traceback (most recent call last):
  File "image_to_video.py", line 2, in <module>
    from diffusers import StableVideoDiffusionPipeline
ImportError: cannot import name 'StableVideoDiffusionPipeline' from 'diffusers' (/home/hzxie/.conda/envs/4DGen/lib/python3.7/site-packages/diffusers/__init__.py)
```

The diffusers installed is `diffusers-0.21.4`.

**Comment by hzxie** [comment]
According to the diffusers's [document](https://huggingface.co/docs/diffusers/en/installation), it requires Python >= 3.8.

But the conda environment file creates an environment with Python 3.7.

**Comment by hzxie** [comment]
Solved this problem by creating Python 3.9 and installing diffusers with the following command

```
pip install torch==1.13.1+cu117 torchvision==0.14.1+cu117 \
       --extra-index-url https://download.pytorch.org/whl/cu117 diffusers["torch"]
```
